packager/app/hls_flags.cc

```diff
@@ -24,3 +24,6 @@ DEFINE_string(hls_playlist_type,
               "VOD, EVENT, or LIVE. This defines the EXT-X-PLAYLIST-TYPE in "
               "the HLS specification. For hls_playlist_type of LIVE, "
               "EXT-X-PLAYLIST-TYPE tag is omitted.");
+DEFINE_int32(hls_media_sequence_number,
+              0,
+              "Number. This defines the initial EXT-X-MEDIA-SEQUENCE value");
```

Comment:
Can you add more background? e.g.

This defines the initial EXT-X-MEDIA-SEQUENCE value, which allows continuous media sequence  across packager restarts. See https://github.com/google/shaka-packager/issues/691 for more backgrounds.

Reply:
Can you also update docs/source/options/hls_options.rst?

Reply:
It is done [here](https://github.com/google/shaka-packager/pull/702/commits/97950bbb93f0b69de0a53f3f428f93e767bab122). Please take a look at it to check if you're ok with what I wrote (english is not my native language).